src/lava/proc/conv_in_time/models.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2021-22 Intel Corporation
```

Comment:
Please change the year for all copyright headers to 2024 for new files, thanks!

Reply:
Thank you for pointing that out, I have fixed them.